extensions/notification/queueProcessor/QueueProcessor.js

```diff
@@ -250,6 +260,15 @@ class QueueProcessor extends EventEmitter {
             return done();
         }
     }
+
+    /**
+     * Checks if queue processor is ready to consume
+     *
+     * @returns {boolean} is queue processor ready
+     */
+    isReady() {
+        return this._consumer && this._consumer.isReady();
```

Comment:
could use the shortcut operator:

```suggestion
        return this._consumer?.isReady();
```

Reply:
the linter version used in backbeat is a bit old, it doesn't like when i write in that syntax